code/modules/clothing/shoes/magboots.dm

```diff
@@ -73,7 +74,7 @@
 	var/state = "disabled"
 	if(src.clothing_flags&NOSLIP)
 		state = "enabled"
-	to_chat(user, "<span class='info'>Its mag-pulse traction system appears to be [state].</span>")
+	to_chat(user, "<span class='info'>Its [anchoring_system] appears to be [state].</span>")
```

Comment:
`appears` should be `appear\s` so that the usage of the s is governed by whether the preceding word is singular or plural.